src/Twilio.php

```diff
@@ -64,7 +64,7 @@ protected function sendSmsMessage(TwilioSmsMessage $message, ?string $to): Messa
     {
         $debugTo = $this->config->getDebugTo();
 
-        if ($debugTo !== null) {
+        if ($debugTo !== null && $debugTo !== "") {
```

Comment:
can we just do use `empty`?

```suggestion
        if (empty($debugTo)) {
```

Reply:
It's possible but please don't invert the logic. It should be `if (! empty($debugTo) ) {`.

Reply:
Yea that's what i meant, oops. Make that change and i'll merge :)

Reply:
:) Changes done, please check it and merge.